openbook-v2-ui/utils/openbook.ts

```diff
@@ -1,44 +1,68 @@
 import {
   findAllMarkets,
-  IDL,
   MarketAccount,
   OPENBOOK_PROGRAM_ID,
   OpenBookV2Client,
 } from "@openbook-dex/openbook-v2";
-import { 
+import {
   ComputeBudgetProgram,
   SystemProgram,
-  PublicKey, 
+  PublicKey,
   TransactionInstruction,
   Connection,
   AddressLookupTableAccount,
   VersionedTransaction,
-  Signer
-} from "@solana/web3.js";
-import {
-  useHookConnection,
-  useFakeProvider,
-} from "../hooks/useOpenbookClient";
-import {
-  Keypair,
-  TransactionMessage
+  Signer,
 } from "@solana/web3.js";
+import { useHookConnection, useFakeProvider } from "../hooks/useOpenbookClient";
+import { Keypair, TransactionMessage } from "@solana/web3.js";
 
 // MAINNET
 // export const RPC = "https://misty-wcb8ol-fast-mainnet.helius-rpc.com/";
-export const RPC = "https://solana-devnet.g.alchemy.com/v2/1CWmcO7kQEa6JVhajg7OHFRQ6ez8B4Hn"; 
+
+export const RPC =
+  "https://solana-devnet.g.alchemy.com/v2/1CWmcO7kQEa6JVhajg7OHFRQ6ez8B4Hn";
+
```

Comment:
This must be protected also and put in the .env file. (The URL contains an access token.)